likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/activityfeed/view/LMFeedActivityFeedFragment.kt

```diff
@@ -0,0 +1,220 @@
+package com.likeminds.feed.android.core.activityfeed.view
+
+import android.os.Bundle
+import android.view.*
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import com.likeminds.feed.android.core.R
+import com.likeminds.feed.android.core.activityfeed.adapter.LMFeedActivityFeedAdapterListener
+import com.likeminds.feed.android.core.activityfeed.model.LMFeedActivityViewData
+import com.likeminds.feed.android.core.activityfeed.viewmodel.LMFeedActivityFeedViewModel
+import com.likeminds.feed.android.core.databinding.LmFeedFragmentActivityFeedBinding
+import com.likeminds.feed.android.core.ui.widgets.headerview.view.LMFeedHeaderView
+import com.likeminds.feed.android.core.utils.*
+import com.likeminds.feed.android.core.utils.LMFeedViewUtils.hide
+import com.likeminds.feed.android.core.utils.LMFeedViewUtils.show
+import com.likeminds.feed.android.core.utils.base.LMFeedBaseViewType
+import com.likeminds.feed.android.core.utils.coroutine.observeInLifecycle
+import kotlinx.coroutines.flow.onEach
+
+open class LMFeedActivityFeedFragment : Fragment(), LMFeedActivityFeedAdapterListener {
+
+    private lateinit var binding: LmFeedFragmentActivityFeedBinding
+    private lateinit var mSwipeRefreshLayout: SwipeRefreshLayout
+
+    private val activityFeedViewModel: LMFeedActivityFeedViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = LmFeedFragmentActivityFeedBinding.inflate(layoutInflater)
+
+        binding.apply {
+            customizeActivityFeedHeaderView(headerViewActivityFeed)
+
+            //set background color
+            val backgroundColor =
+                LMFeedStyleTransformer.activityFeedFragmentViewStyle.backgroundColor
+            backgroundColor?.let { color ->
+                root.setBackgroundColor(
+                    ContextCompat.getColor(
+                        requireContext(),
+                        color
+                    )
+                )
+            }
+
+            return root
+        }
+    }
+
+    protected open fun customizeActivityFeedHeaderView(headerViewActivityFeed: LMFeedHeaderView) {
+        headerViewActivityFeed.apply {
+            setStyle(LMFeedStyleTransformer.activityFeedFragmentViewStyle.headerViewStyle)
```

Comment:
check lint

Reply:
it looks good as per Android Studio linting